CONTRIBUTING.md

```diff
@@ -0,0 +1,643 @@
+
+However, there are a few things that we have to fix manually.
+
+- "Use `super()` instead of `super(__class__, self)`" \
+   Most architectures use the old style of calling `super()` to support Python 3.7. Spandrel does not support Python 3.7, so you can just replace `super(__class__, self)` with `super()`.
+- Unused variables \
+   Researchers are trying to push what is possible and often not too concerned with code quality. You will often find unused variables which our linters do not like. Instead of removing those variables, prefix them with `_`. In general, don't try to clean up the code (yet). You do not want to accidentally change the architecture code and then wonder why the official models aren't working.
+- Type errors \
+   Some of them can be fixed by adding type annotations, but it's often best to just use a `# type: ignore` comment to suppress the error (for now).
+- Removing `**kwargs` \
+   The main class of your architecture might have an used parameter `**kwargs` or `**ignorekwargs`. Remove it. This parameter means that typoing a parameter name will not result in an error, and it will just silently ignore the parameter. This is the cause of much frustration and many bugs, so save yourself and remove the parameter. \
+   If the parameter is used, leave it for now.
+
+In general, try to change the original code as little as possible in this stage. We can clean up the code after we have fully added support for the architecture and have tests to make sure that we didn't break anything.
+
+Finally, some model files contain a `if __name__ == "__main__":` section to run the model as a script. We do not ever want to run the model as a script, so we can just remove this section.
+
+In the case of DITN, we just ignore a few type errors and remove [this section](https://github.com/yongliuy/DITN/blob/3438e429c0538ee5061a7cfca587df0c4097703f/models/DITN_Real.py#L249).
+
+### Step 1.2: Fixing dependencies
+
+The model files might have dependencies to external packages. We generally try to keep the number of dependencies for spandrel as low as possible, so we want to remove any dependencies that are not strictly necessary.
+
+We allow models to have the following external dependencies: `torch`, `torchvision`, `numpy`, and `einops`.
+
+`timm` is a special dependency for us, because we have vendored the most important code from `timm`. So instead of using the `timm` package, we should use the vendored code from `src/spandrel/architectures/__arch_helpers/timm/`.
+
+In the case of DITN, its model file has no external dependencies, except for `torch` and `einops`, so we are done.
+
+### Step 2: Adding a `load` method
+
+With the architecture code in place, we can start integrating it into spandrel. The first step is to define a `load` function. This function will take a state dict, detect the hyperparameters of the architecture (=the input arguments of the architecture's Python class), and return a loaded model.
```

Comment:
Is that = a typo or intentional?

Reply:
It's intentional. I'm explaining what hyperparameters are.